src/Optimization/hiopKKTLinSysSparse.cpp

```diff
@@ -473,77 +471,77 @@ namespace hiop
   bool hiopKKTLinSysCompressedSparseXDYcYd::
   solveCompressed(hiopVector& rx, hiopVector& rd, hiopVector& ryc, hiopVector& ryd,
```

Comment:
I aligned opening brace statements with the closing braces. I suggest we leave it like this. Otherwise, it is difficult to see the logic of conditional statements here.

Reply:
@nychiang, your commit looks good, thanks!